prophyc/generators/python.py

```diff
@@ -148,3 +149,8 @@ class PythonGenerator(GeneratorBase):
     top_level_translators = {
         ".py": _PythonTranslator
     }
+
+    def check_nodes(self, _):
+        init_file = os.path.join(self.output_dir, "__init__.py")
```

Comment:
That's controversial :) User can easily do that themselves when wanting to generate a package. Some users might not want to generate a package - it might be harmful to force generating __init__.py on them.